docs/extend/modular-resources/create.md

```diff
@@ -59,15 +59,17 @@ The following example module registers a modular resource implementing Viam's bu
 
 The Go code for the custom model (<file>mybase.go</file>) and module entry point file (<file>main.go</file>) are adapted from the full demo modules available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources).
```

Comment:
```suggestion
The Go code for the custom model (<file>mybase.go</file>) and module entry point file (<file>main.go</file>) is adapted from the full demo modules available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources).
```
sorry I realize I accidentally suggested the opposite earlier

Reply:
fixed